src/components/report-viewer-tab.js

```diff
@@ -189,8 +197,10 @@ export const ReportViewerTab = ({
                                 inputProps={{
                                     'aria-label': 'primary checkbox',
                                 }}
-                                onChange={(e) => handleChangeValue(e)}
-                                disabled={disabled}
+                                onChange={(e) => handleChangeNodeOnlySwitch(e)}
+                                disabled={
+                                    disabled || rootNodeId === currentNode?.id
```

Comment:
If we disable the switch on the root node, maybe we also should force the "checked" value to either true or false if the current node is root, because at the moment, it is possible to manipulate the `parent-nodes-report?nodeOnlyReport=` parameter in the REST query by setting the switch to true or false in another node.

That should not be possible, and the results seem to always be the same, but that should not be allowed anyway.

Reply:
I think it is not usefull